src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -1,4 +1,4 @@
-package io.sentry.demos.example;
+package io.sentry.samples.example;
```

Comment:
Application.java was not appearing in the stacktrace because Sentry sees errors from 'sentry' or demo packages as noisy. Manoel suggested to rename to 'samples'. It worked.